README.md

```diff
@@ -158,8 +158,47 @@ dbt docs serve
 ```
 </details>
 
+<details>
+<summary>GitHub Codespaces / Dev Containers </summary>
+
+#### Steps
+
+1. Ensure you have Codespaces enabled for your GitHub organization or turned on as a beta feature if you're an individual user
+2. Click the green **Code** button on near the top right of the page of this repo's homepage (you may already be on it)
+3. Instead of cloning the repo like you normally would, intead select the **Codespaces** tab of the pop out, then "Create codespace on `duckdb`" (`duckdb` happens to be the main branch, else if #29 is not yet merged make a codespace on the `add_codespace` branch)
```

Comment:
we should probably make this change before merging. @sungchun12 are these docs good enough for an initial merge?
```suggestion
3. Instead of cloning the repo like you normally would, intead select the **Codespaces** tab of the pop out, then "Create codespace on `duckdb`"
```